Package.swift

```diff
@@ -0,0 +1,16 @@
+// swift-tools-version:5.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+import PackageDescription
+
+let package = Package(
+    name: "RxWebKit",
+    products: [
+        .library(name: "RxWebKit", targets: ["RxWebKit"])
+    ],
+    dependencies: [
+        .package(url: "https://github.com/ReactiveX/RxSwift", from: "5.0.1")
```

Comment:
```suggestion
        .package(url: "https://github.com/ReactiveX/RxSwift", from: "5.0.1"),
        .package(url: "https://github.com/Quick/Quick.git", from: "2.2.0"),
        .package(url: "https://github.com/Quick/Nimble.git", .branch("master"))
]
```

Reply:
master for now: https://github.com/mattgallagher/CwlPreconditionTesting/issues/16